atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -582,6 +582,14 @@ def get_build(self, metadata):
             else:
                 extra['image']['parent_build_id'] = parent_id
 
+        # Append isolated build flag
+        try:
+            isolated = str(metadata['labels']['isolated']).lower() == 'true'
+        except (IndexError, AttributeError, KeyError):
+            isolated = False
+        self.log.info("build is isolated: {}".format(isolated))
```

Comment:
Would be nice to change both this and the koji_import instance of this line to avoid the landscape.io warning:
https://landscape.io/github/projectatomic/atomic-reactor/2109/modules/atomic_reactor/plugins/exit_koji_promote.py#L590